subsys/mgmt/mcumgr/grp/img_mgmt/src/img_mgmt.c

```diff
@@ -159,6 +159,12 @@ int img_mgmt_active_slot(int image)
 	/* Multi image does not support DirectXIP currently */
```

Comment:
so this comment should be removed now

Reply:
The comment is still accurate, this is a "hack" for one specific mode